dag/energy.yml

```diff
@@ -86,7 +86,7 @@ steps:
   data://garden/energy/2024-06-20/energy_mix:
     - data://garden/demography/2023-03-31/population
     - data://garden/energy_institute/2024-06-20/statistical_review_of_world_energy
-    - data://garden/wb/2024-03-11/income_groups
+    - data://garden/wb/2025-07-01/income_groups
```

Comment:
Is this needed here, @pabloarosado? You don't seem to call it, and it runs fine without it

Reply:
Good catch, it's not needed.